test/view/example_test.js

```diff
@@ -0,0 +1,5 @@
+var App = require('./../../app/example');
+console.log(App);
```

Comment:
You'll know you're getting somewhere when you can get some DOM output in a karma test here.